sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2274,6 +2274,26 @@ val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.
       .integerConf
       .createWithDefault(1024)
 
+  val ENABLE_DELTA_LOW_SHUFFLE_MERGE =
+    conf("spark.rapids.sql.delta.lowShuffleMerge.enabled")
+    .doc("Option to turn on the low shuffle merge for Delta Lake.")
+    .booleanConf
+    .createWithDefault(false)
+
+  val DELTA_LOW_SHUFFLE_MERGE_SCATTER_DEL_VECTOR_BATCH_SIZE =
+    conf("spark.rapids.sql.delta.lowShuffleMerge.deletion.scatter.max.size")
+    .doc("Option to set max batch size when scattering deletion vector")
+    .integerConf
+    .createWithDefault(4096)
+
+  val DELTA_LOW_SHUFFLE_MERGE_DEL_VECTOR_BROADCAST_MAX_COUNT =
+    conf("spark.rapids.sql.delta.lowShuffleMerge.deletionVector.broadcast.max.count")
+    .doc("Option to set max broadcast count of low shuffle merge deletion vector. If the " +
+        "detected deletion vector row count is larger than this value, low shuffle merge will be " +
+        "disabled.")
+    .longConf
+    .createWithDefault(600000000)
```

Comment:
It would be good to have a comment explaining how this was computed for future reference.  Since it was sensitive to the broadcast size, should this be derived from the configured broadcast size?